### PR TITLE
[ADD] dms_partner_attachment_link: add module

### PR DESCRIPTION
**Pull request by dmitriypaulov**
This module adds "DMS Attachments" smart button to the partner's form leading to a set of related, stored as attachments DMS files

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Which is the difference with https://github.com/OCA/dms/tree/15.0/dms_attachment_link ?

**Comment by pedrobaeza** [comment]
And why not using `dms_field` for this?

**Comment by dmitriypaulov** [comment]
> Which is the difference with https://github.com/OCA/dms/tree/15.0/dms_attachment_link ?

The difference is that dms_attachment_link adds a wizard for files quick creation from records of specific models. This module just adds a shortcut to see partner related files. I hope that is clear, please let me know if I'm wrong

**Comment by pedrobaeza** [comment]
Well, not really clear the usefulness of this, but it's OK, don't worry.

File: dms_partner_attachment_link/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        :rtype: dict
+        """
+        self.ensure_one()
+        action = self.env.ref("dms.action_dms_file").read()[0]
+        action.update(
+            {
+                "domain": [
+                    ("res_id", "=", self.id),
+                    ("res_model", "=", "res.partner"),
+                    ("storage_id.save_type", "=", "attachment"),
```
Comment (ivs-cetmix) [inline dms_partner_attachment_link/models/res_partner.py]: What is this `storage_id` domain condition for?
Shouldn't all the files be shown no matter where they are stored?

**Review by ivs-cetmix — CHANGES_REQUESTED** [review]
Rather a question yet...

File: dms_partner_attachment_link/models/res_partner.py (hunk 1 above)
Comment (dmitriypaulov) [reply dms_partner_attachment_link/models/res_partner.py]: From the task description: "Keep in mind that we can find related partners only for files that are stored in the Attachment Storages"

Comment (dmitriypaulov) [reply dms_partner_attachment_link/models/res_partner.py]: @ivs-cetmix see also readme/DESCRIPTION.md file

**Review by victoralmau — COMMENTED** [review]
Although I understand the need for this module, doing so will always make a new one necessary for almost any model and I think that is not a good approach.

My approach to this would be more transversal (maybe add it even in the `dms` module), add an action automatically in the form view of a record (only if there is some document: folder or file) called "View Documens" (or similar) and that when clicked we go to the "Files" action (kanban view) with the folder of the record selected. what do you think?

**Pull request closed without merging**